packages/types/src/organization.ts

```diff
@@ -32,6 +32,7 @@ export interface OrganizationResource extends ClerkResource {
   membersCount: number;
   pendingInvitationsCount: number;
   publicMetadata: OrganizationPublicMetadata;
+  adminDeleteEnabled: boolean;
```

Comment:
❓ Shouldn't we parse this setting in /core/resources/Organization.ts?

Reply:
🤦‍♂️ i for some reason did not push this change - ty!